opentelemetry-otlp/src/lib.rs

```diff
@@ -36,12 +36,11 @@
 //!
 //! fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
 //!     // First, create a OTLP exporter builder. Configure it as you need.
-//!     let otlp_exporter = opentelemetry_otlp::new_exporter().tonic();
+//!     let otlp_exporter = opentelemetry_otlp::SpanExporter::builder().with_tonic().build()?;
```

Comment:
nit - remove the `new_pipeline` reference from doc comment above (line 27)

Reply:
updated the doc comment to be more accurate for these changes.